FTBFS: error: ‘scalars_name’ was not declared in this scope

I tried to compile the latest version on master and it failed with the following error:
```
[ 51%] Building CXX object Applications/src/CMakeFiles/calculate-element-wise.dir/calculate-element-wise.cc.o
/home/francois/workspace/MIRTK/Applications/src/calculate-element-wise.cc: In function ‘int main(int, char**)’:
/home/francois/workspace/MIRTK/Applications/src/calculate-element-wise.cc:672:41: error: ‘scalars_name’ was not declared in this scope
       const char *output_scalars_name = scalars_name;
                                         ^
Applications/src/CMakeFiles/calculate-element-wise.dir/build.make:62: recipe for target 'Applications/src/CMakeFiles/calculate-element-wise.dir/calculate-element-wise.cc.o' failed
make[2]: *** [Applications/src/CMakeFiles/calculate-element-wise.dir/calculate-element-wise.cc.o] Error 1
CMakeFiles/Makefile2:2220: recipe for target 'Applications/src/CMakeFiles/calculate-element-wise.dir/all' failed
make[1]: *** [Applications/src/CMakeFiles/calculate-element-wise.dir/all] Error 2
```

--- a/Applications/src/calculate-element-wise.cc
+++ b/Applications/src/calculate-element-wise.cc
@@ -686,8 +686,6 @@ int main(int argc, char **argv)
           #else
             Warning("Output scalars array name argument of -output option ignored");
           #endif
-        } else {
-          output_scalars_name = scalars_name;
         }
       }
       #if MIRTK_Image_WITH_VTK

--- a/Applications/src/calculate-element-wise.cc
+++ b/Applications/src/calculate-element-wise.cc
@@ -668,9 +668,11 @@ int main(int argc, char **argv)
         ops.push_back(UniquePtr<Op>(new Write(fname, datatype, attr)));
       #endif
     } else if (OPTION("-o") || OPTION("-out") || OPTION("-output")) {
-      const char *fname               = ARGUMENT;
-      const char *output_scalars_name = scalars_name;
-      int         dtype               = datatype;
+      const char *fname = ARGUMENT;
+      int         dtype = datatype;
+      #if MIRTK_Image_WITH_VTK
+        const char *output_scalars_name = scalars_name;
+      #endif
       if (HAS_ARGUMENT) {
         const char *arg = ARGUMENT;
         dtype = ToDataType(arg);
@@ -678,7 +680,15 @@ int main(int argc, char **argv)
           cerr << "Invalid -out data type " << arg << endl;
           exit(1);
         }
-        if (HAS_ARGUMENT) output_scalars_name = ARGUMENT;
+        if (HAS_ARGUMENT) {
+          #if MIRTK_Image_WITH_VTK
+            output_scalars_name = ARGUMENT;
+          #else
+            Warning("Output scalars array name argument of -output option ignored");
+          #endif
+        } else {
+          output_scalars_name = scalars_name;
+        }
       }
       #if MIRTK_Image_WITH_VTK
         ops.push_back(UniquePtr<Op>(new Write(fname, dtype, attr, dataset, scalars_name, output_scalars_name)));

